src/components/ae-button-group/ae-button-group.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="ae-button-group" :class="{ shadow }">
+    <slot />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'ae-button-group',
+  props: {
+    /**
+     * Apply shadow to the button group, applicable only for round buttons
+     * shadow has a 32px roundness
+     */
+    shadow: Boolean,
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@import '../../styles/variables/colors';
+
+.ae-button-group {
+  display: flex;
+  border-radius: 32px;
+  overflow: hidden;
+
+  &.shadow {
+    box-shadow: 0 0 16px $color-shadow-alpha-15;
+  }
+
+  /deep/ .ae-button {
+    border-radius: 0;
+    border: 0;
+    box-shadow: none;
+    flex-grow: 1;
+
+    &.round {
+      border-left-width: 1px;
+      border-right-width: 1px;
+      border-style: solid;
```

Comment:
https://app.zeplin.io/project/59e4cf99fc0bb2f99a89551b/screen/5b97e915ac40214afbbdc55b
Actually, I can't see any border between buttons on this screen.

Reply:
@davidyuk fixed.